volume/service/service.go

```diff
@@ -0,0 +1,129 @@
+package service // import "github.com/docker/docker/volume/service"
+
+import (
+	"context"
+
+	"github.com/docker/docker/pkg/idtools"
+	"github.com/docker/docker/pkg/plugingetter"
+	"github.com/docker/docker/volume"
+	"github.com/docker/docker/volume/drivers"
+	"github.com/docker/docker/volume/local"
+	"github.com/pkg/errors"
+)
+
+type vs interface {
```

Comment:
This seems to be a very different abstraction from what we are calling services currently (ex: ImageService, and RegistryService) which implement higher level logic.

Most of this interface is implemented by the `VolumeStore`, right? 

I would think a `VolumeService` would be a type that implements `api/server/router/volume.Backend`, as well as whatever is used by other services. That way we remove the need for a monolithic daemon package.

I like the changes in this PR, but it seems that we're losing the ability to create a VolumeService to replace the daemon methods for volumes.

Could this continue to be a `VolumeStore`, and we split out a service later?

Reply:
We could call it that still... but that (replace daemon methods) is indeed the intent, it's just yet more change to add in this large PR, of course can add a commit to make that happen.

Reply:
Made this change.